.circleci/config.yml

```diff
@@ -166,21 +161,42 @@ jobs:
     <<: *executor
     steps:
       - *attach_workspace
-      - *import_image
+      - *import_final_image
+      - checkout
+      - *install_pytest
       - run: |
           mkdir ~/testresults
-          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v ~/testresults:/output signalfx-agent-dev pytest -n8 --junitxml=/output/integration_tests.xml -m "not packaging and not installer and not k8s" tests
+          mkdir -p /tmp/scratch
+          mkdir bundle
+          docker run --rm -v `pwd`/bundle:/tmp/bundle quay.io/signalfx/signalfx-agent-dev:latest \
+              bash -c 'cp -r /bin /etc /jvm /lib /lib64 /neopy /plugins /run /var /tmp/bundle/'
+          export AGENT_BIN=`pwd`/bundle/bin/signalfx-agent
+          sudo setcap CAP_NET_RAW=+eip $AGENT_BIN
+          export TEST_SERVICES_DIR=`pwd`/test-services
+          sudo -E pytest \
```

Comment:
Choose between either doing sudo here or doing the setcap -- they shouldn't both be necessary.